diff --git a/lua/chadrc.lua b/lua/chadrc.lua
index 986a835..0aefefe 100644
--- a/lua/chadrc.lua
+++ b/lua/chadrc.lua
@@ -8,7 +8,7 @@ M.ui = {
    italic_comments = false,
 
    -- theme to be used, to see all available themes, open the theme switcher by <leader> + th
-   theme = "onedark",
+   theme = "blossom",
 
    -- theme toggler, toggle between two themes, see theme_toggleer mappings
    theme_toggler = {
@@ -47,7 +47,7 @@ M.options = {
    hidden = true,
    ignorecase = true,
    insert_nav = true, -- navigation in insertmode
-   mapleader = " ",
+   mapleader = ",",
    mouse = "a",
    number = true,
    -- relative numbers in normal mode tool at the bottom of options.lua
@@ -89,7 +89,7 @@ M.plugin_status = {
    neoscroll = true, -- smooth scroll
    telescope_media = false, -- see media files in telescope picker
    truezen = false, -- no distraction mode for nvim
-   vim_fugitive = false, -- git in nvim
+   vim_fugitive = true, -- git in nvim
    vim_matchup = true, -- % magic, match it but improved
 }
 
