diff --git a/lua/chadrc.lua b/lua/chadrc.lua
index 986a835..0aefefe 100644
--- a/lua/chadrc.lua
+++ b/lua/chadrc.lua
@@ -8,7 +8,7 @@ M.ui = {
    italic_comments = false,
 
    -- theme to be used, to see all available themes, open the theme switcher by <leader> + th
-   theme = "onedark",
+   theme = "blossom",
 
    -- theme toggler, toggle between two themes, see theme_toggleer mappings
    theme_toggler = {
@@ -47,7 +47,7 @@ M.options = {
    hidden = true,
    ignorecase = true,
    insert_nav = true, -- navigation in insertmode
-   mapleader = " ",
+   mapleader = ",",
    mouse = "a",
    number = true,
    -- relative numbers in normal mode tool at the bottom of options.lua
@@ -89,7 +89,7 @@ M.plugin_status = {
    neoscroll = true, -- smooth scroll
    telescope_media = false, -- see media files in telescope picker
    truezen = false, -- no distraction mode for nvim
-   vim_fugitive = false, -- git in nvim
+   vim_fugitive = true, -- git in nvim
    vim_matchup = true, -- % magic, match it but improved
 }
 
diff --git a/lua/core/autocmds.lua b/lua/core/autocmds.lua
index c74ef95..ac70626 100644
--- a/lua/core/autocmds.lua
+++ b/lua/core/autocmds.lua
@@ -15,3 +15,9 @@ vim.cmd [[ autocmd BufEnter,BufWinEnter,FileType,WinEnter * lua require("core.ut
 -- vim.cmd [[ au BufReadPost * if expand('%:p') !~# '\m/\.git/' && line("'\"") > 1 && line("'\"") <= line("$") | exe "normal! g'\"" | endif ]]
 -- File extension specific tabbing
 -- vim.cmd [[ autocmd Filetype python setlocal expandtab tabstop=4 shiftwidth=4 softtabstop=4 ]]
+
+-- autoformat on save
+vim.cmd [[ augroup fmt
+  autocmd!
+  autocmd BufWritePre * undojoin | Neoformat
+augroup END ]]
